src/main/java/hu/akarnokd/rxjava2/parallel/ParallelOrderedMerge.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2016-2017 David Karnok
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package hu.akarnokd.rxjava2.parallel;
+
+import java.util.Comparator;
+
+import org.reactivestreams.Subscriber;
+
+import hu.akarnokd.rxjava2.operators.BasicMergeSubscription;
+import io.reactivex.Flowable;
+import io.reactivex.parallel.ParallelFlowable;
+
+final class ParallelOrderedMerge<T> extends Flowable<T> {
```

Comment:
Please add a short description of this class here, include `@since 0.17.9` and `@author X` where `X` is you.